File: FinalSubmission02/cpp/Assignment02.cpp
#include <iostream>

int main() {
    int num1, num2; // Declaration of variable
    std::cout<<"Please enter number 1: "; // Printing on terminal
    std::cin>>num1; // User input   
    std::cout<<"Please enter number 2: "; // Printing on terminal
    std::cin>>num2; // User input   
    //Conditional Statements
    if(num1>num2){
        std::cout<<"The greatest number is "<<num1;
        }
        else if(num1<num2){
        std::cout<<"The greatest number is "<<num2;
        }
    else
        std::cout<<"Both numbers are equal";
return 0;
}

File: FinalSubmission02/Assignment02.java
//Import util package
import java.util.Scanner;
//Declaration of class
public class Assignment02 {

    //Declaration of main method
    public static void main(String[] args) {
        
        //Declaration of variables
        int num1, num2;
        //Calling Scanner class for user input
        Scanner input = new Scanner(System.in);
        //Prompt for user input on terminal
        System.out.println("Please enter Number 01: ");
        //Number 1 User input
        num1 = input.nextInt();
        //Prompt for user input on terminal
        System.out.println("Please enter Number 02: ");
        //Number 2 user input
        num2 = input.nextInt();

            //Putting conditional or selection statements
            if (num1>num2){
                System.out.println("Number 1 is greater than Number 2!");
            }
            else if(num1<num2){
                System.out.println("Number 2 is greater than Number 1!");
            }
            else
                System.out.println("Number 1 is equal to Number 2!");
        //Closing Scanner class    
        input.close();
    }
}